src/Str.php

```diff
@@ -97,6 +99,14 @@ public static function asEventMethod(string $eventName): string
         return sprintf('on%s', self::asClassName($eventName));
     }
 
+    public static function asFilePath(string $value): string
+    {
+        $value = Container::underscore(trim($value));
+        $value = str_replace('\\', '/', $value);
```

Comment:
this anticipates the possibility to generate controllers in subnamespace
(which is forbidden right now, but doesn't have to be really)